### ISSUE TITLE
Window rule property not applying

### ISSUE THREAD

**Issue by dglava** [labwc/labwc#944]
I cannot get window rule properties to apply. This is the error I am getting:
```
00:00:00.001 [ERROR] [../src/config/rcxml.c:111] expect <action name=""> element first. nodename: 'skipwindowswitcher' content: 'yes'
```

My config file contains this:
```
<windowRules>
  <windowRule identifier="firefox" skipWindowSwitcher="yes"/>
</windowRules>
```

Am I doing something wrong?
Labwc version: 0.6.3

**Comment by johanmalm** [comment]
Not sure. I've tried with that config and I don't get a warning.

Are you definitely running a clean `0.6.3` version?

Could you post your whole config?

**Comment by dglava** [comment]
Yes, I compiled 0.6.3 myself on Arch Linux and labwc -v outputs 0.6.3.
Link to my config: https://gist.github.com/dglava/6b9d0aeaf0a07b6c0b9eb377d06d3e20

**Comment by stefonarch** [comment]
```
    <windowRule identifier="firefox" skipWindowSwitcher="yes"/>
  </windowRules>
```
one "s" to much:
```

    <windowRule identifier="firefox" skipWindowSwitcher="yes"/>
  </windowRule>
```

I (somehow) remember openbox refusing reload with syntax errors, I did quite some in labwc which didn't complain about.

**Comment by dglava** [comment]
I think that's the closing tag of the general *windowRules* segment? Not the closing tag of the single *windowRule*.

What I am trying to accomplish is to have both skipWindowSwitcher and the toggleDecorations action active for Firefox. I have come up with this, but it's giving me the same complaint and it's appearing in the window switcher (alt+tab menu). The *toggleDecorations* action seems to be working though.

```
  <windowRules>
      <windowRule identifier="firefox" skipWindowSwitcher="yes" >
          <action name="toggleDecorations" />
    </windowRule>
  </windowRules>
```

**Comment by johanmalm** [comment]
Your syntax seems fine.

skipWindowSwitcher was only implemented after 0.6.3 so that’s probably the issue :smile:

Would you be able to try with the master branch?

**Comment by dglava** [comment]
Hehe, yes, that fixed it. Thank you, it's working properly now. :+1:
